dask_cloudprovider/cloudprovider.yaml

```
@@ -117,3 +117,13 @@ cloudprovider:
     image: "ubuntu-20.04" # Operating System image to use
     docker_image: "daskdev/dask:latest" # docker image to use
     bootstrap: true # It is assumed that the OS image does not have Docker and needs bootstrapping. Set this to false if using a custom image with Docker already installed.
+
+  fly:
+    token: null # API token for interacting with the Fly API
+    region: "lax" # Region to launch Droplets in
+    vm_size: "shared-cpu-1x" # Droplet size to launch, default is 1GB RAM, 1 vCPU
+    image: "daskdev/dask:latest" # Operating System image to use
```

This should use GHCR. We should probably update the others too.
```suggestion
    image: "ghcr.io/dask/dask:latest" # Operating System image to use
```